Basic support for WeMo Insight Switch

austinheap opened the pull request:
Tried to follow the design pattern, let me know if this is okay!

df-arif commented:
Hi there, thanks for your interest in this project and for the pull request as well. I don't have a Wemo Insight switch to test your changes. Can you please confirm that this works for you? Also please update the README to indicate the support for Wemo Insight switch. Just add it to the list of supported devices after #3 (Wemo Switch). Once you confirm that it works and update the README, I will merge your PR. Thanks!

austinheap commented:
Hey Arif, thanks for the great library! I've been struggling to find *anything* that works with the Insight. I've tested the following functions on my Insights successfully: `getParams`, `On`, `Off`, `getBinaryState`. I couldn't get `GetPower`, `GetTodayKWH`, or any of the other Insight-specific commands working, but those are all derivable from GetInsightParams.

I've updated the `README.md` and added an example.

df-arif commented:
Thanks for the changes and the example. As long as the methods in the DeviceInterface works for now it all good.
